packages/auth/demo/README.md

```diff
@@ -57,6 +57,7 @@ You can optionally clear the cache and rebuild using:
 ```bash
 cd auth/demo
 rm -rf node_modules yarn.lock
+yarn
```

Comment:
I usually do `yarn build:deps && yarn install`

Reply:
I've done in the other order, but makes sense to install deps and then build like you have in this PR.